kvbc/test/sparse_merkle_storage/db_adapter_unit_test.cpp

```diff
@@ -105,7 +106,7 @@ bool hasStaleIndexKeysSince(const std::shared_ptr<IDBClient> &db, const Version
   auto iter = db->getIteratorGuard();
   const auto key = iter->seekAtLeast(DBKeyManipulator::genStaleDbKey(version)).first;
   if (!key.empty() && DBKeyManipulator::getDBKeyType(key) == EDBKeyType::Key &&
-      DBKeyManipulator::getKeySubtype(key) == EKeySubtype::Stale &&
+      DBKeyManipulator::getKeySubtype(key) == EKeySubtype::ProvableStale &&
```

Comment:
Maybe that function should be renamed, because it only checks for `ProvableStale` keys.

Reply:
Done